packages/fx-core/templates/plugins/resource/hosting/bicep/botFunction/config.template.bicep

```diff
@@ -0,0 +1,12 @@
+// Get existing app settings for merge
+var botFunctionCurrentAppSettings = list('${provisionOutputs.botFunctionOutput.value.botWebAppResourceId}/config/appsettings', '2021-02-01').properties
+
+// Merge TeamsFx configurations to Function resources
+module teamsFxBotFunctionConfig '\{{PluginIdPlaceholder.Configuration.botFunction.path}}' = {
```

Comment:
Rename this folder to `botFunction`, and add `bot` keyword in modules' name.
Otherwise it conflict with function plugin.

Reply:
Thanks for identify this issue, and I am also thinking the similar issue in our email thread with Tim. You can share with me more information offline.